packages/fxa-auth-server/docs/service_notifications.md

```diff
@@ -216,5 +216,5 @@ so other services probably shouldn't use this.
 - `uid`: User id.
 - `subscriptionId`: Subscription id.
 - `isActive`: Boolean indicating whether the subscription is active.
-- `productName`: Product name.
+- `productId`: Product id.
```

Comment:
@bbangert, note I decided to propagate this change through the service notification boundary and into the event broker too. My thinking was just for consistency, although of course I don't have a full picture of what's involved for downstream consumers so if you think that's a step too far, I'm happy to roll it back.